src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_utils.py

```diff
@@ -6,6 +6,8 @@
 from azure.cli.core.util import CLIError
 from azure.cli.core.commands.parameters import get_resources_in_subscription
 
+from azure.mgmt.containerregistry.v2017_03_01.models import SkuTier
```

Comment:
@troydai since @djyou says there is no need to expose multiple api-versions to the users, it's fine to import a specific version. It won't work on Azure Stack but I assume that was not intended anyway.